client/js/components/statement/fragmentList/Fragment.vue

```diff
@@ -604,7 +604,7 @@ export default {
     },
 
     fragmentDocumentTitle () {
-      if (hasOwnProp(this.fragment, 'document')) {
+      if (hasOwnProp(this.fragment.document, 'document')) {
```

Comment:
this expression would check if `this.fragment.document` has a property `document`.. is this the expected structure, like `this.fragment.document.document`?

Reply:
its `this.fragment.document` , but it didnt work this way. I changed it to one line without hasOwnProp https://github.com/demos-europe/demosplan-core/pull/2830/commits/f4802c5e9c2e67b42794a2962cd996bca9ee9dd8